jsonld.js/src/test/scala/org/w3/banana/jsonldjs/io/JsonLdJsParserTest.scala

```diff
@@ -0,0 +1,39 @@
+package org.w3.banana
+package jsonldjs
+package io
+
+import zcheck.SpecLite
+import scalajs.concurrent.JSExecutionContext.Implicits.runNow
+import org.w3.banana.plantain._
+
+object JsonLdJsParserTest extends SpecLite {
```

Comment:
Do we have a generic test? I mean with zcheck. As far as I know nobody has ported it. We have already discussed what that entails and that is a lot of work. For now, this is testing that the facades are correctly wired.